### ISSUE TITLE
Exclude more non-text characters from color-contrast

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2125]
Here's an example of non-text characters that should be marked for review instead of failed by the color-contrast rule:

Source: https://act-rules.github.io/rules/afw4f7#passed-example-8

```html
<p style="color: #000; background: #666;">
----=====++++++++___________***********%%%%%%%%%%%±±±±@@@@@@@@
</p>
```

**Comment by padmavemulapati** [comment]
Verified in axe-coconut(v4.4.1.08515), yes for the non-text charecters there is no failure , not even pass, its showing in the best-practice.

![image](https://user-images.githubusercontent.com/47409579/77651330-a4dbd480-6f92-11ea-9dac-26aa6ed4a81e.png)

**Comment by padmavemulapati** [comment]
Ran through axe-core latest  code, which is  having fix changes for (color-contrast), found that, for non-text charecter color-contrast is not failing but is in inapplicable.
![image](https://user-images.githubusercontent.com/47409579/77746808-871a7800-7043-11ea-9d9f-f23f22ff1759.png)

**Comment by straker** [comment]
No docs needed
